app/simulator/slack/page.tsx

```diff
@@ -0,0 +1,269 @@
+"use client";
```

Comment:
Delete this page. We only want this new functionality within the admin dashboard